src/seraphis_crypto/sp_legacy_proof_helpers.cpp

```diff
@@ -77,14 +77,6 @@ static std::size_t highest_bit_position(std::size_t num)
 }
 //-------------------------------------------------------------------------------------------------------------------
 //-------------------------------------------------------------------------------------------------------------------
-void append_clsag_to_transcript(const rct::clsag &clsag_proof, SpTranscriptBuilder &transcript_inout)
-{
-    transcript_inout.append("s", clsag_proof.s);
```

Comment:
`LegacyClsagProof` replaces `rct::clsag` as a transaction component. Only difference is no `I` (key image) field since already stored in legacy input image.